src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -562,8 +571,7 @@ class IpexCommunicationService extends AgentService {
     }
   }
 
-  @OnlineOnly
-  async joinMultisigOffer(said: string): Promise<void> {
+  private async joinMultisigOffer(said: string): Promise<void> {
     const exn = await this.props.signifyClient.exchanges().get(said);
```

Comment:
This assumes the `said` is of `/multisig/exn` but it needs to be the SAID of the IPEX offer that's embedded. So we can directly fetch that and not look in the embeds.

Reply:
I understand the problem you are referring to. But if I replace the SAID value (multisig/exn) with the direct SAID of ipex/apply message then I won't have enough data to call the multisigOfferAcdcFromApply() function inside. Although it is possible to update linkedGroupRequests